source/standards/git.html.md.erb

```diff
@@ -18,6 +18,16 @@ developers on your project. This includes:
 * any future developers (including yourself) who want to see why a change was
   made
 
+  Taking the time to write a good commit message is important for a few reasons.
```

Comment:
The double-space at the beginning of this line should be removed.

Reply:
Well spotted!